test/e2e/common.go

```diff
@@ -221,3 +225,61 @@ func buildKustomizeManifest(source string) ([]byte, error) {
 	}
 	return resources.AsYaml()
 }
+
+func executeSSHCommand(client *ssh.Client, command string) (string, error) {
+	session, err := client.NewSession()
+	if err != nil {
+		return "", fmt.Errorf("failed to create SSH session: %v", err)
+	}
+	defer session.Close()
+
+	output, err := session.CombinedOutput(command)
+	if err != nil {
+		return "", fmt.Errorf("failed to execute command '%s': %v", command, err)
+	}
+
+	return string(output), nil
+}
+
+// ParseFileSystemOutput parses the given filesystem output string and checks if booted from disk
+func ParseFileSystemOutput(output string) bool {
```

Comment:
I would suggest renaming to clarify the output
```suggestion
// HasRootOnDisk parses the output from df -h and checks if the root filesystem is on a disk (as opposed to tmpfs).
func HasRootOnDisk(dfOutput string) bool {
```